devsetup/scripts/gen-edpm-bmaas-kustomize.sh

```diff
@@ -26,6 +26,8 @@ NETCONFIG_CR_FILE=${NETCONFIG_CR_FILE:-netconfig.yaml}
 DNSMASQ_CR_URL=${DNSMASQ_CR_URL:-https://raw.githubusercontent.com/openstack-k8s-operators/infra-operator/main/config/samples/network_v1beta1_dnsmasq.yaml}
 DNSMASQ_CR_FILE=${DNSMASQ_CR_FILE:-dnsmasq.yaml}
 BMH_CR_FILE=${BMH_CR_FILE:-bmh_deploy.yaml}
+EDPM_REGISTRY_URL=${EDPM_REGISTRY_URL:"quay.io/podified-antelope-centos9"}
```

Comment:
I mentioned the earlier comment at the wrong place. Put this in the make file too.

Reply:
Also missing "-"

Reply:
Thank you for finding that. I have removed the vars from here and move it to Makefile at proper target.